core/src/test/resources/com/squarespace/template/plugins/platform/f-image-3.html

```diff
@@ -8,11 +8,10 @@
     "title": "This should not be used, as the info.altText field should override",
     "originalSize": 1000,
     "assetUrl": "/foo/bar.jpg",
-    "mediaFocalPoint": {"x": 0.3, "y": 0.7}
-  },
-  "info": {
+    "mediaFocalPoint": {"x": 0.3, "y": 0.7},
     "altText": "baz"
-  }
+  },
```

Comment:
Rely on the enum's integer value instead of the name

Reply:
Done